Day2/index.js

```diff
@@ -0,0 +1,120 @@
+var skillpoint = 50;
+var str = 0;
+var def = 0;
+var int = 0;
+var lux = 0;
+
+$(() => {
+
+  /* STR */
+  $('#dec-str').on('click', (arguments) => {
+    if( str > 0 && skillpoint < 50) {
+      str--; skillpoint++;
+      document.getElementById("str-percent").style = "width: " + str*2 + "%";
```

Comment:
This approach is fine, but since we used jQuery, we should keep the consistency by using whatever jQuery provided for us.
This code is just changing the `css`, so in jQuery we could write `$('#str-percent').css('width', str*2 + '%')`